types/index.d.ts

```diff
@@ -57,6 +58,7 @@ declare var Connection: {
     CELL_2G: string;
     CELL_3G: string;
     CELL_4G: string;
+    CELL_5G: string;
     CELL: string;
     NONE: string;
 }
```

Comment:
```suggestion
}

```

Add new line at end of file.